modules/release-notes/ref-release-notes-breaking-changes.adoc

```diff
@@ -4,4 +4,60 @@
 
 This section lists breaking changes in {product} {product-version}.
 
-None
\ No newline at end of file
+
+[id="removed-functionality-rhdhbugs-2000"]
+== Using an optional Secret named `dynamic-plugins-npmrc` to configure a custom npm repository in the Operator is removed
+
+Previously, an optional Secret named `dynamic-plugins-npmrc` could be configured in the Operator as a custom npm repository.
```

Comment:
```suggestion
Previously, the Operator could automatically detect and mount an optional Secret named `dynamic-plugins-npmrc` to configure a custom npm registry.
```

To rephrase probably, but the previous sentence seems a bit misleading otherwise IMO, as the Secret is actually not configured in the Operator.